client/components/editor-standalone/save-handling.ts

```diff
@@ -77,5 +77,11 @@ export const handleEmbeddedPlannings = async(
 };
 
 export const embeddedPlanningHasUnsavedChanges = () => {
-    return getEmbeddedPlanningExposed(EDITOR_TYPE.INLINE).some((x) => x?.hasUnsavedChanges?.());
+    const planningsExposed = getEmbeddedPlanningExposed(EDITOR_TYPE.INLINE);
+
+    if (planningsExposed.length > 0) {
```

Comment:
`return (planningsExposed ?? []).some((x) => x.hasUnsavedChanges());` is cleaner and avoid this extra condition